services/scan/schema.sql

```diff
@@ -1,3 +1,18 @@
+create table election (
+  -- enforce singleton table
+  id integer primary key check (id = 1),
+  election_definition text not null,
+  precinct_selection text,
+  is_test_mode boolean not null default true,
+  skip_election_hash_check boolean not null default false,
+  polls_state text not null default "polls_closed_initial",
+  marginal_mark_threshold_override real,
+  definite_mark_threshold_override real,
+  cvrs_backed_up_at datetime,
+  scanner_backed_up_at datetime,
+  created_at timestamp not null default current_timestamp
+);
+
 create table batches (
   batch_number integer primary key autoincrement,
   id varchar(36) unique,
```

Comment:
Could put foreign keys on `batches` to `election`, but if we really think it's a singleton table, then that seems like a waste.

Reply:
You can do cascade deletion with a foreign key. See [this example](https://github.com/votingworks/vxsuite/blob/ea1602678a988eba239c69237a696303a6924ff0/services/admin/schema.sql#L17-L18). And yes, I do think we should add these foreign keys.

Reply:
SGTM.